phone/src/apps/contacts/components/ContactsApp.tsx

```diff
@@ -9,10 +9,11 @@ import { ContactPage } from './views/ContactsPage';
 import { ContactsThemeProvider } from '../providers/ContactsThemeProvider';
 import PersonAddIcon from '@mui/icons-material/PersonAdd';
 import Fab from '@mui/material/Fab';
-import { useHistory, useLocation } from 'react-router-dom';
+import { useHistory, useLocation } from 'react-router';
```

Comment:
Why do we switch to react-router here?

Reply:
To be honest I'm not sure how this change became part of this PR, as only my changes

Reply:
Please change it back :)

Reply:
This has been fixed!